src/solvers/generic.ml

```diff
@@ -32,13 +32,8 @@ end
 (* move this to some other place! *)
 module ExtendInt (B:Analyses.VarType) : Analyses.VarType with type t = B.t * int =
 struct
-  type t = B.t * int
+  type t = B.t * int [@@deriving eq, ord]
   let relift x = x
-  let compare ((u1,u2):t) (v1,v2) =
-    match Stdlib.compare u2 v2 with
-    | 0 -> B.compare u1 v1
-    | n -> n
-  let equal ((u1,u2):t) (v1,v2) = u2=v2 && B.equal u1 v1
```

Comment:
Actually this place also used to compare second components first, so I'll have to look what the reason is.

Reply:
This `ExtendInt` is actually used by some `MoreVars` SLR variants, so I'll revert it just in case.